src/app/modules/form/admin-form/__tests__/admin-form.service.spec.ts

```diff
@@ -3072,4 +3075,221 @@ describe('admin-form.service', () => {
       )
     })
   })
+
+  describe('updateOptionsToRecipientsMap', () => {
+    describe('validation of options to recipients map', () => {
+      it('should return error if options to recipients map has options that are not in the field options', async () => {
```

Comment:
@KenLSM to add validation when `fieldType` is not `Dropdown`

Reply:
done in [d400c77](https://github.com/opengovsg/FormSG/pull/7804/commits/d400c77d4e85d7a13f9c9377c0333a1bee5c5f9f)